### ISSUE TITLE
Research GUI is ambiguous

### ISSUE THREAD

**Issue by DylanHsu** [Subsentient/wz2100legacy#11]
The research GUI is ambiguous. I propose that on mouse rollover, the user is shown a non-exhaustive list of icons which a particular research leads to. There may be a better way to make the research GUI more intuitive.

**Comment by wei2912** [comment]
Good idea, that's one of the difficulties I had when i first started playing (and till now as well).
